ios15环境下，同wifi下做本地调试，打开网页执行window.location.href 会破坏vite运行环境导致项目终止运行

只在ios15环境下，同wifi下调试，会出现此问题。
一旦执行window.location.href ，立马项目终止运行，其他版本ios 和 安卓手机都可以正常运行。
报错代码如下。



/Users/xxxxxxxxxxxxxxx/node_modules/vite/dist/node/chunks/dep-2c03f3f9.js:53050
      else bu.unmask(buffer, mask);
              ^

TypeError: bu.unmask is not a function
    at unmask (/Users/jingxiaotian/Documents/***路径***/node_modules/vite/dist/node/chunks/dep-2c03f3f9.js:53050:15)
    at Receiver.getData (/Users/jingxiaotian/Documents/xxxxxxxxxxxxxxx/node_modules/vite/dist/node/chunks/dep-2c03f3f9.js:54012:25)
    at Receiver.startLoop (/Users/jingxiaotian/Documents/***路径***/node_modules/vite/dist/node/chunks/dep-2c03f3f9.js:53808:22)
    at Receiver._write (/Users/jingxiaotian/Documents/***路径***/node_modules/vite/dist/node/chunks/dep-2c03f3f9.js:53743:10)
    at writeOrBuffer (_stream_writable.js:353:12)
    at Receiver.Writable.write (_stream_writable.js:303:12)
    at Socket.socketOnData (/Users/jingxiaotian/Documents/***路径***/node_modules/vite/dist/node/chunks/dep-2c03f3f9.js:55875:37)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:302:12)
    at readableAddChunk (_stream_readable.js:278:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! project@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the project@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jingxiaotian/.npm/_logs/2021-06-18T02_33_54_321Z-debug.log

--- a/packages/vite/rollup.config.js
+++ b/packages/vite/rollup.config.js
@@ -161,14 +161,10 @@ const createNodeConfig = (isProduction) => {
             replacement: `: eval('require'),`
           }
         }),
-      // Optional peer deps of ws. Native deps that are mostly for performance.
-      // Since ws is not that perf critical for us, just ignore these deps.
-      ignoreDepPlugin({
-        bufferutil: 1,
-        'utf-8-validate': 1
-      }),
       commonjs({
         extensions: ['.js'],
+        // Optional peer deps of ws. Native deps that are mostly for performance.
+        // Since ws is not that perf critical for us, just ignore these deps.
         ignore: ['bufferutil', 'utf-8-validate']
       }),
       json(),
@@ -256,26 +252,6 @@ function shimDepsPlugin(deps) {
             )
           }
         }
-      }
-    }
-  }
-}
-
-/**
- * @type { (deps: Record<string, any>) => import('rollup').Plugin }
- */
-function ignoreDepPlugin(ignoredDeps) {
-  return {
-    name: 'ignore-deps',
-    resolveId(id) {
-      if (id in ignoredDeps) {
-        return id
-      }
-    },
-    load(id) {
-      if (id in ignoredDeps) {
-        console.log(`ignored: ${id}`)
-        return ''
       }
     }
   }

